demos/input/index.html

```diff
@@ -164,6 +199,12 @@
             <div id="validate" class="btn pointer">Validate</div>
         </div>
     </div>
+
+    <datalist id="data-list">
```

Comment:
I am a little bit worried about that. That means that the components are relying on something from the outside. Shouldn't we pass the options to the component and let it render the datalist on its own?